formats/schemas/january-2015.txt

```diff
@@ -0,0 +1,166 @@
+Changes For January 2015
+========================
+
+
+The list of the key changes for the January 2015 major release of the
+ome-xml data model. This schema release will tie in with the Bio-Formats
+5.1 release.
+
+The new major release of the schema has a new namespace and all version
+numbers are reset to 1. As a major release, any file that validated
+correctly using the last major release will probably not validate
+correctly using this new release. Some files that failed to validate
+before will now be valid. It is important to update any file readers and
+writers to understand the changes.
+
+The version number of all schema files is now 1.
+
+This schema uses the new namespace:
+
+::
+
+    http://www.openmicroscopy.org/Schemas/[NameSpaceTitle]/2015-01/
+
+For the OME schema
+
+::
+
+    http://www.openmicroscopy.org/Schemas/OME/2015-01/
+
+and the schema file is located at
+
+::
+
+    http://www.openmicroscopy.org/Schemas/OME/2015-01/ome.xsd
+
+Overview of changes
+-------------------
+
+- This release is preparation for the release of Bio-Formats 5.1 and OMERO
+  5.1.
+- This version sees the introduction of a major new system of specifying units
+  for the values stored as lengths, times, pressures, angles, temperatures,
+  electric potentials (voltages), powers and frequencies.
+- The annotation points have been greatly expanded and some rationalised.
+  Objects in the model are now directly annotatable or are in a 1 to 1
+  relationship with an object that is.
+
+
+BinaryFile
+^^^^^^^^^^
+
+- There are no significant changes to this component.
+
+OME
+^^^
+
```

Comment:
In this section , Addition/Removal/Adding/Define should all be past tense since this has already happened.  I.e. Added/Removed/Defined.